cli/README.md

```diff
@@ -8,25 +8,27 @@ As a beginning developer it's important to get familiar with it, as it will teac
 
 Follow along with the following video (the video is for the MacOS terminal, so if you are on windows you can use your Git Bash and run the same commands):
 
-{% hyf-youtube src="https://www.youtube.com/watch?v=5XgBd6rjuDQ" %}
+{% hyf-youtube src=“https://www.youtube.com/watch?v=5XgBd6rjuDQ” %}
```

Comment:
Can you revert it back to the regular quotes? From my experience, using a different type of symbol can mess up things

Reply:
Sure, I can revert to regular quotes, but this will open a smaller YouTube player, which isn’t as as nice-looking. The current quote was part of the original code and is used in the HYF book, displaying a full-width video player. This repository uses this style of quotes for YouTube videos throughout.

If you prefer, I can use can use regular quotes.

**or** 

it is possible to use Markdown with embedded HTML for a responsive video link:

```html
<div style="width: 100%; max-width: 100vw;">
     <a href="https://www.youtube.com/watch?v=5XgBd6rjuDQ" target="_blank">
         <img src="https://img.youtube.com/vi/5XgBd6rjuDQ/0.jpg" alt="Video Terminal Basics for Beginners" style="width: 100%; height: auto;"/>
     </a>
</div>
```
**or** we can leave it as it is. 

Whatever you prefer.